tests/console/verify_bcache_partitioning.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate bcache in writethrough
+# The backing device is /home and it is attached to the caching device which is the second
+# disk. Ususally the caching device is a fast device (ssd). In this scenario a rotational device is used.
+# This doesnt give any significant performance advantages but it is better to test bcache with this setup
+# than use bcache without caching(which yast provides a option if we want to do so)
+# Scenarios covered:
+# - Verify that certain values are set correctly in a bcache setup after installation;
```

Comment:
s/;/./

Reply:
i am pretty sure that you can find this in other modules as well. And matter of taste here. Lets avoid this

Reply:
a semicolon at the end of a line looks like a typo, though. IMHO a comment is a comment not a line in a programming language which ends line with semicolon. I guess that there are plenty of grammatical errors in the repo but no need for repeat them, I guess.

Reply:
i will change it but this goes against the flexibility which quite confuse me. if there are not restrictions we should not review those kind of typos. I have fixed the rest of the grammatical errors but this is something else entirely.

Reply:
f35918e67

Reply:
Flexibility is that you can do things differently, for example, use different kind of commands for your validation. You can share your view and have and open discussion giving your arguments and reasoning. 
For this particular case, grammatical errors are usually addressed in PRs. Of course if you insist on not changing it I would merge anyway as it is a minor thing but it is a public repo and it is for our own good to correct those mistakes.